salt/output/nested.py

```diff
@@ -39,9 +38,13 @@ class NestDisplay(object):
     Manage the nested display contents
     '''
     def __init__(self):
-        self.colors = salt.utils.get_colors(
+        self.__dict__.update(
```

Comment:
I'd probably put a comment of what this ends up doing-- not sure if everyone knows about __dict__ ;)